app/res/layout/blank_screen_login.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="fill_parent"
+    android:orientation="vertical">
+
+    <AutoCompleteTextView
+        android:id="@+id/edit_username"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:inputType="text"
+        android:visibility="gone" />
+
+    <EditText
+        android:id="@+id/edit_password"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:inputType="textPassword"
+        android:visibility="gone" />
+
+</LinearLayout>
```

Comment:
The entire startup process, which includes installing the .ccz (so we are technically on the `SetupActivity` screen) and restoring a user from the local restore file (so we are technically on the `LoginActivity` screen) looks like this
![screenshot_20160512-131427](https://cloud.githubusercontent.com/assets/3723143/15223708/cf16dc4a-1843-11e6-9e24-3e52d518d29a.png)